### PR TITLE
Add info to docs about concurrent session limits

### PR DESCRIPTION
**Pull request by matthewhelmke**
This parallels a [change in the Grafana Cloud docs](https://github.com/grafana/website/pull/3504) which was based on a kind and appreciated message from support. The limit does not apply to Grafana OSS, just to Enterprise and Cloud, as far as any of us know.

Tagging both @osg-grafana and @achatterjee-grafana as reviewers as I'm not certain who owns these docs, @oddlittlebird is out, and I don't want to step on toes. I'm just trying to help out with a quick change. :)

### CODE REVIEW

File: docs/sources/enterprise/license-restrictions.md
Comment (achatterjee-grafana) [inline docs/sources/enterprise/license-restrictions.md:28]:
```suggestion
Sometimes it is useful to log in to an account from multiple locations concurrently. With Grafana Enterprise, accounts are limited to three concurrent sessions.
```

**Review by achatterjee-grafana — APPROVED** [review]
Added a copy-edit suggestion.

**Comment by osg-grafana** [comment]
@mjseaman for visibility

**Comment by mjseaman** [comment]
Concurrent session limits will not be enforced until v7.5. - are you planning to update this along with the 7.5 release?

**Comment by matthewhelmke** [comment]
I'm afraid I don't know the answer to that. This is enforced in cloud (and I own those docs) and I was told it was also true here. I'll defer to the main docs maintainers for changes as appropriate.

**Comment by achatterjee-grafana** [comment]
At the very least, please add a note this will be available in 7.5 or revisit this after the 7.5 release. Thanks!

**Comment by matthewhelmke** [comment]
This was merged seven days ago. I'll create a new PR with a note about 7.5.

**Pull request merged**
